docs/configuration/storage-config.md

```diff
@@ -146,3 +146,5 @@ storage:
     flavor: minio
     endpoint: http://127.0.0.1:9000
 ```
+
+Note: `default_index_root_uri` or index URIs do not include the endpoint, you should set it as a typical S3 path such as `s3://indexes`.
```

Comment:
Just a minor addition in the doc as I saw 2 users making this mistake, that is setting the URI as `s3://127.0.0.1:9000/indexes`